src/services/FavouritesArticlesService.ts

```diff
@@ -1,116 +1,42 @@
-    createdAt: '2021-07-08T18:09:35.898Z',
-    updatedAt: '2021-07-08T18:09:35.898Z',
-    description: 'texto sobre que es el articulo',
-    tagList: [],
-    author: {
-      username: 'churry',
-      bio: null,
-      image: '',
-      following: false
-    },
-    favorited: false,
-    favoritesCount: 0
-  },
-  {
-    title: 'Articulo de pruebaa',
-    slug: 'articulo-de-pruebaa-ddyrum',
-    body: 'Descripcion del articulo',
-    createdAt: '2021-07-06T19:26:45.145Z',
-    updatedAt: '2021-07-06T19:26:45.145Z',
-    description: 'texto sobre que es el articulo',
-    tagList: [],
-    author: {
-      username: 'username7602',
-      bio: null,
-      image: 'https://static.productionready.io/images/smiley-cyrus.jpg',
-      following: false
-    },
-    favorited: false,
-    favoritesCount: 0
-  },
-  {
-    title: 'Articulo de pruebaa',
-    slug: 'articulo-de-pruebaa-akw3g9',
-    body: 'Descripcion del articulo',
-    createdAt: '2021-07-06T19:23:47.176Z',
-    updatedAt: '2021-07-06T19:23:47.176Z',
-    description: 'texto sobre que es el articulo',
-    tagList: [],
-    author: {
-      username: 'username5380',
-      bio: null,
-      image: 'https://static.productionready.io/images/smiley-cyrus.jpg',
-      following: false
-    },
-    favorited: false,
-    favoritesCount: 0
-  },
-  {
-    title: 'Articulo de pruebaa',
-    slug: 'articulo-de-pruebaa-1dk3ry',
-    body: 'Descripcion del articulo',
-    createdAt: '2021-07-06T19:14:10.187Z',
-    updatedAt: '2021-07-06T19:14:10.187Z',
-    description: 'texto sobre que es el articulo',
-    tagList: [],
-    author: {
-      username: 'username6733',
-      bio: null,
-      image: 'https://static.productionready.io/images/smiley-cyrus.jpg',
-      following: false
-    },
-    favorited: false,
-    favoritesCount: 0
-  }
-];
+import { create } from 'apisauce';
+import Config from 'react-native-config';
+import { FAVORITE_PATH } from '@constants/favoriteServiceConstants';
+import { CurrentUser } from '@interfaces/authInterfaces';
 
-const ArticlesService = {
-  getMockFavourites: () =>
-    Promise.resolve({
-      ok: true,
-      problem: null,
-      originalError: null,
-      data: FAVOURITED_MOCK
+// const FAVOURITED_MOCK = [
+//   {
+//     title: 'Argentina vs Brasil',
+//     slug: 'argentina-vs-brasil-p5q4e4',
+//     body: 'asdasdasdsadsadsad',
+//     createdAt: '2021-07-08T18:40:16.960Z',
+//     updatedAt: '2021-07-08T18:40:16.960Z',
+//     description: 'Sábado final Copa América ',
+//     tagList: [],
+//     author: {
+//       username: 'zeque11',
+//       bio: 'Fanático de Racing Club de Avellanada, el club más grande de Argentina. Único heptacampeón y glorioso por su gente y su historia.',
+//       image: 'https://www.elterritorio.com.ar/img/1/289/2020060820580200_1.jpg',
+//       following: false
+//     },
+//     favorited: false,
+//     favoritesCount: 1
+//   },
+// ];
+const baseURL = Config.API_BASE_URL;
+const api = create({ baseURL });
```

Comment:
Can you use api from config/api?

Reply:
I tried again and it's work idk why, change import to '@config/api';